gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobuflayer.cpp

```diff
@@ -617,7 +617,7 @@ OGRGeometry *OGRFlatGeobufLayer::readGeometry(const Feature *feature)
 {
     auto pXy = feature->xy();
     if (pXy == nullptr)
-        CPLError(CE_Fatal, CPLE_AppDefined, "readGeometry: Geometry has no coordinates");
+        CPLError(CE_Failure, CPLE_AppDefined, "readGeometry: Geometry has no coordinates");
```

Comment:
you'll get a nullptr dereference on the next line

Reply:
Ah yes sorry, should have realized I have to rework the error handling when changing this.